src/main/webapp/i18n/en/programmingExercise.json

```
@@ -142,7 +142,9 @@
                     "updated": "The test cases were updated successfully.",
                     "couldNotBeUpdated": "The following test cases could not be updated: {{testCases}}.",
                     "weightSumError": "The sum of all test case weights is 0 or below. Please adjust at least one test case weight to be greater than 0",
-                    "showInactive": "Show inactive test cases"
+                    "showInactive": "Show inactive test cases",
+                    "settingNull": "Test case {{testCase}} must not have settings that are null.",
+                    "settingNegative": "Test case {{testCase}} must not have settings set to negative numbers."
```

```suggestion
                    "settingNegative": "Test case {{testCase}} must not have settings set to negative values."
```
As in the German translation.